wicket-core/src/main/java/org/apache/wicket/markup/head/CssHeaderItem.java

```diff
@@ -169,6 +171,39 @@ public static CssReferenceHeaderItem forReference(ResourceReference reference,
 	}
 
 	/**
+	 * Creates a {@link CssReferenceHeaderItem} for the given reference.
+	 *
+	 * <strong>Warning</strong>: the conditional comments don't work when injected dynamically with
```

Comment:
Conditional comments are deprecated since a long time in IE (since 2012, I think).
I think we should not care about them.